lib/voxpupuli/acceptance/spec_helper_acceptance.rb

```diff
@@ -13,7 +13,7 @@ def configure_beaker(modules: :metadata, &block)
 
   require_relative 'fixtures' if modules == :fixtures
 
-  unless ENV['BEAKER_provision'] == 'no'
+  unless ENV['BEAKER_PROVISION'] == 'no'
```

Comment:
@evgeni pointed out this can be considered breaking. Let's fix it up and check for both.